README.md

```diff
@@ -14,12 +14,104 @@ I-Talent is an internal web-application that would enable employees to share inf
 
 # Getting started
 
-You'll need to create three files to define the envirnment variables, one at the root of the project and two in the `env` folder. To get those variables, ask the current developers. 
+You'll need to create three files to define the environment variables, one at the root of the project and two in the `env` folder. To get those variables, ask the current developers.
 
 [Docker](https://www.docker.com/) and [Visual Studio Code](https://code.visualstudio.com/) are technically optional tools for development, but are highly recommended! A list of useful Visual Studio Code extensions are listed [here](https://github.com/CDH-Studio/I-Talent/wiki/Tech-stack#useful-visual-studio-code-extensions).
 
 In the root of the project, to start developing the application use the [Docker Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) or run the following command:
 
+## Downloading and Configuring Dependencies
+
+Download "Visual Studio Code" for Windows
+Run the installer
+Once installed open Visual Studio Code
+Go to the Extensions icon (or press Ctrl+Shift+X)
+Install "Prettier", and "Docker"
+
+Download "Docker Desktop" for Windows
+Run the "Docker Desktop Installer"
+On configuration screen press "Ok"
+Once it finishes "Unpacking files..." and "Installing..." it should say the "Installation succeeded"
+Press on "Close and restart" (Note: This will restart Windows)
```

Comment:
you can just say "Download and install Docker Desktop" instead of going into detail about the installation process